### PR TITLE
avoid leaving a condition that does not perform any action

### PR DESCRIPTION
**Pull request by devrchancay**
avoid leaving a condition that does
not perform any action
 Changes to be committed:
	modified:   src/index.js

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -20,19 +20,19 @@ export default function createStore(worker) {
...
-		if (typeof data!=='object') {}
-		else if ('pop' in data) {
-			if (data.length===1) {
+
+		if (typeof data === 'object') {
+			if ('pop' in data && data.length === 1) {
 				process(data[0]);
 			}
-			else {
+			else if (data.length > 0) {
```
Comment (developit) [inline src/index.js]: This extra check adds unnecessary weight to the final bundle.

**Comment by devrchancay** [comment]
And in this way? 
How I see it the for loop will run until when 'data' has an element. 

```javascript
function handleMessage({ data }) {
	if (typeof data === 'object') {
		if ('pop' in data) {
			for (let i=0; i<data.length; i++) process(data[i]);
		}
		else {
			process(data);
		}
	}
}
```

**Comment by developit** [comment]
The current implementation is optimized for single-element arrays, since that's the 99% case.

**Comment by devrchancay** [comment]
I suppose that in these cases it is valid, leave the code how it is.

**Pull request closed without merging**